app-modules/audit/tests/AuditListenerTest.php

```diff
@@ -39,24 +39,26 @@
 use AidingApp\Audit\Settings\AuditSettings;
 use AidingApp\ServiceManagement\Models\ServiceRequest;
 
-test('Audit logs are only created if the Model is set to be Audited by audit settings', function () {
+test('Audit logs are only created if the Model is not set to be excluded from Auditing by audit settings', function () {
     asSuperAdmin();
 
+    $serviceRequest = ServiceRequest::factory()->make();
+
     $auditSettings = resolve(AuditSettings::class);
 
-    $auditSettings->audited_models = [];
+    $auditSettings->audited_models_exclude = [$serviceRequest->getMorphClass()];
 
     $auditSettings->save();
 
-    $serviceRequest = ServiceRequest::factory()->create();
-
     expect($serviceRequest->audits)->toHaveCount(0);
 
-    $auditSettings->audited_models[] = $serviceRequest->getMorphClass();
+    $auditSettings->audited_models_exclude[] = [];
```

Comment:
This should be resetting it to a blank array, not setting a new value as a blank array

```suggestion
    $auditSettings->audited_models_exclude = [];
```